client/apps/game/src/ui/modules/navigation/top-left-navigation.tsx

```
@@ -36,53 +44,80 @@ const structureIcons: Record<string, JSX.Element> = {
+        return a.name.localeCompare(b.name);
+      });
+  }, [favorites, structures, structureNameChange]);
 
   const filteredStructures = useMemo(() => {
     if (!searchTerm) return structuresWithFavorites;
     return structuresWithFavorites.filter((structure) =>
-      structure.name.toLowerCase().includes(searchTerm.toLowerCase()),
+      structure.name
+        .toLowerCase()
+        .normalize("NFD")
+        .replace(/[\u0300-\u036f]/g, "")
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)